R-package/README.md

```diff
@@ -0,0 +1,564 @@
+
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/RGF)](http://cran.r-project.org/package=RGF)
+[![Travis-CI Build Status](https://travis-ci.org/mlampros/RGF.svg?branch=master)](https://travis-ci.org/mlampros/RGF)
+[![codecov.io](https://codecov.io/github/mlampros/RGF/coverage.svg?branch=master)](https://codecov.io/github/mlampros/RGF?branch=master)
+[![Downloads](http://cranlogs.r-pkg.org/badges/grand-total/RGF?color=blue)](http://www.r-pkg.org/pkg/RGF)
+
+
+## RGF (Regularized Greedy Forest)
+<br>
+
+
+The **RGF** package is a wrapper of the [Regularized Greedy Forest (RGF)](https://github.com/RGF-team/rgf_python) *python* package, which also includes a [Multi-core implementation (FastRGF)](https://github.com/baidu/fast_rgf). More details on the functionality of the RGF package can be found in the [blog-post](http://mlampros.github.io/2018/02/14/the_RGF_package/) and in the package Documentation. 
+
+<br>
+
+**UPDATE 26-07-2018**: A [Singularity image file](http://mlampros.github.io/2018/07/26/singularity_containers/) is available in case that someone intends to run *RGF* on Ubuntu Linux (locally or in a cloud instance) with all package requirements pre-installed. This allows the user to utilize the *RGF* package without having to spend time on the installation process.
+
+<br>
+
+**References:**
+
+[Rie Johnson and Tong Zhang, Learning Nonlinear Functions Using Regularized Greedy Forest](https://arxiv.org/abs/1109.0887)
+
+https://github.com/RGF-team/rgf/tree/master/RGF
+
+https://github.com/RGF-team/rgf/tree/master/FastRGF
+
+https://github.com/RGF-team/rgf/tree/master/python-package
+
+
+<br>
+
+### **System Requirements**
+
+<br>
+
+* Python (2.7 or >= 3.4)
+
+
+<br>
+
+All modules should be installed in the default python configuration (the configuration that the R-session displays as default), otherwise errors will occur during the *RGF* package installation (**reticulate::py_discover_config()** might be useful here). 
+
+<br>
+
+#### **Debian/Ubuntu/Fedora**    [ Python 2.7 ]
```

Comment:
I think that now you can just give a link to the `rgf_python` installation guide and write here only those notes which are related to the **R-package**, like `After a successful rgf-python installation the user should open an R session and give the following *reticulate* command `, `**NOTE : CURRENTLY THE PACKAGE ON WINDOWS CAN BE USED ONLY FROM THE COMMAND LINE (cmd)** ` and others. Generally, it's bad idea to duplicate the information which is already can be found in the neighbor folder; providing a link is enough. Doing this you'll be free from fetching changes from upstream guides.

Reply:
I think that it's good that you commented on this case and I believe that now @fukatani should also participate in this pull request / discussion. 
My opinion to this matter is that the installation instructions of RGF and FastRGF should be more detailed.
I say this because it took me personally many hours to install RGF on different Operating Systems and different python versions and I think that users (like me for instance) should not spend hours or even days in order to at least test the rgf_python or a future version of the R-package.

Reply:
Yeah, of cource!

We can together refine/enhance installation instructions of RGF/FastRGF/rgf_python, but it's out of the scope of this PR.

Reply:
No, I think it's related because we first have to refine / enhance the installation instructions of the RGF / FastRGF / rgf_python and then we should proceed with the update / modification of the installation instructions in the R-package.

Reply:
Unfortunately, we won't mix up different things in one PR making it extremely large.

But reworking the R package guide after enhancing other is good point. So, I suggest you only fix stuff connected with migration (paths/urls), but remained refactoring will be processed in next PR.